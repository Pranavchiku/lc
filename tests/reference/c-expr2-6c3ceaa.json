{
    "basename": "c-expr2-6c3ceaa",
    "cmd": "lc --no-color --show-c {infile} -extra-arg=\"-Isrc/runtime/include\"",
    "infile": "tests/expr2.c",
    "infile_hash": "11cb114715d67a4763454167abc900089b2119ecf18a980d18a30322",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "c-expr2-6c3ceaa.stdout",
    "stdout_hash": "e6191aea238f10f620726b70b3fc7aaf8dc513951f81c57aa02ff210",
    "stderr": "c-expr2-6c3ceaa.stderr",
    "stderr_hash": "618edec93f196191cdcaa86eaa9eff5724164cc3a30c09c436d87d90",
    "returncode": 0
}