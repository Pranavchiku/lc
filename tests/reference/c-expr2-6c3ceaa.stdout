#include <inttypes.h>

#include <stdlib.h>
#include <stdbool.h>
#include <stdio.h>
#include <string.h>
#include <lfortran_intrinsics.h>

int32_t _xx_lcompilers_changed_main_xx();



// Implementations
int32_t _xx_lcompilers_changed_main_xx()
{
    int32_t __return_var;
    int32_t x;
    x = (2 + 3)*5;
    printf("%d\n", x);
    __return_var = 0;
    return __return_var;
}

int main(int argc, char* argv[])
{
    _lpython_set_argv(argc, argv);
    int32_t exit_code;
    exit_code = _xx_lcompilers_changed_main_xx();
    return 0;
}
